src/sass/media-queries.scss

```diff
@@ -1,26 +0,0 @@
-@media screen and (max-width: 768px) {
- #page-wrapper {
-      #content {
-          .page-section {
-              & > .tab-menu {
-                  ul.tm-ul {
-                      li {
-                          &.active {
-                              a {
-                                  background: none;
-                              }
-                          }
-                      }
-                  }
-              }
-          }
-      }
-  }
-
-  #sell_popup_container {
```

Comment:
have we fixed this somewhere else? It was added so that we show our sell popup as 100% on mobile and 7 -inch tablet